sdk/confidentialledger/azure-resourcemanager-confidentialledger/src/test/java/com/azure/resourcemanager/confidentialledger/ConfidentialLedgerManagementTestBase.java

```diff
@@ -68,6 +69,7 @@ public void setupManager() {
         } else if (getTestMode() == TestMode.PLAYBACK) {
             ledgerManager = ConfidentialLedgerManager
                 .configure()
+                .withDefaultPollInterval(Duration.ofMillis(10))
```

Comment:
Thanks.

... one more reason to avoid RECORD test.

Reply:
@alzimmermsft It occured to me that DPG lib might have LRO tests as well. How does data-plane handle it?

Reply:
Many of the data-plane tests end up having a method for setting the poll interval based on playback or live testing but there are methods available on `TestBase` which handle setting this: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core-test/src/main/java/com/azure/core/test/TestBase.java#L286

Ideally, those would be used more often.

Reply:
I see, so "setPlaybackSyncPollerPollInterval" should be called after getting the `SyncPoller` and before calling it to get final result, by dev in test code. Is it correct?